Form-Controls/styles.css

```diff
@@ -17,55 +17,56 @@ h1{
 }
 
 input{
-  width: 400px;
+  width: 100%;
+  max-width: 350px;
   height: 15px;
-  font-size: 20px;
+  font-size: 18px;
   padding: 10px;
-  border: 0px solid white;
   border-radius: 8px;
   margin-top: 20px;
   margin-bottom: 20px;
-  margin-left: 20px;
 }
 select{
-  width: 400px;
+  width: 100%;
+  max-width: 150px;
   height: 40px;
-  font-size: 20px;
-  padding: 10px;
-  border: 0px solid white;
-  border-radius: 2px;
+  font-size: 18px;
+  padding: 2px;
+  border-radius: 8px;
   margin-top: 5px;
   margin-bottom: 20px;
-  margin-left: 85px;
 }
 
 
 form{
-  width: 45%;
+  width: 100%;
+  max-width: 500px;
   margin-left:auto;
   margin-right: auto;
   border:1px solid white;
- border-radius: 3px;
+ border-radius: 18px;
  padding: 20px;
  
 }
 
 .button{
   height: 40px;
-  width: 150px;
+  width: 100%;
+  max-width: 100px;
   font-size: 20px;
   margin-bottom: 10px;
   padding: 2px;
-  border: 0px solid #222222;
   border-radius: 3px;
   background: lightslategray; 
   color: black;
   cursor: pointer;
-  margin-left: 30%;
 }
 #btn-align{
   display: flex;
-
+  justify-content: center;
+}
+#btn-space{
+  margin-right: 8px;
```

Comment:
instead of creating new `id` selector, you can add `gap: 8px` to  `#btn-align`.
You may read about gap property here - https://www.w3schools.com/cssref/css3_pr_gap.php

Reply:
Thank you for this new knowledge.